core/src/main/scala/org/apache/spark/ui/jobs/JobsTab.scala

```diff
@@ -30,6 +30,7 @@ private[ui] class JobsTab(parent: SparkUI, store: AppStatusStore)
   extends SparkUITab(parent, "jobs") {
 
   val sc = parent.sc
+  val conf = parent.conf
```

Comment:
is this needed?

Reply:
Yes. Without this, we can't access `SparkConf` from `AllJobsPage` and `JobPage`.
We can't do `parent.parent.conf` from those pages.
`StageTab` has the same code for `StagePage`.